inventory/group_vars/all.yml

```diff
@@ -82,6 +83,11 @@ openshift_cluster_content:
     namespace: labs-ci-cd
     tags:
     - nexus
+    post_steps:
+    - role: "labs-ci-cd/roles/configure-nexus"
+      vars:
+        nexus_name: "nexus"
+        nexus_project: "labs-ci-cd"
```

Comment:
I'd like to remove the vars - I do not think they are needed in this case, we can use defaults and see below about project name